lib/galaxy/main_config.py

```diff
@@ -17,7 +17,11 @@
 
 
 def default_relative_config_paths_for(app_name: str) -> List[str]:
-    return [f"config/{app_name}.yml", f"config/{app_name}.ini", "universe_wsgi.ini", f"config/{app_name}.yml.sample"]
+    paths = [f"config/{app_name}.yml", f"config/{app_name}.ini", "universe_wsgi.ini"]
+    # Do not load sample config for galaxy
+    if app_name != "galaxy":
```

Comment:
What are the other possible app_names here?

Reply:
I suppose it's `tool_shed` and `reports`. The `reports` app does not load defaults from the schema (it was in "maintenance-only" mode at the time when we changed how configuration is processed for galaxy; but maybe it would make sense to fix reports too). The `tool_shed` app loads from tool shed's config schema, but I didn't want to touch it right now as it's being worked on. Once this is done for the galaxy app, we can include tool_shed, then maybe reports - once step at a time should be easiest/safest here, I think.